AdGoBye/Indexer.cs

```diff
@@ -324,21 +324,36 @@ public static void PatchContent(Content content)
         Logger.Information("Processing {ID} ({path})", content.Id, content.Path);
         foreach (var plugin in PluginLoader.LoadedPlugins)
         {
-            var pluginApplies = plugin.Instance.PluginType() == EPluginType.Global;
-            if (!pluginApplies && plugin.Instance.PluginType() == EPluginType.ContentSpecific)
+            try
             {
-                var ctIds = plugin.Instance.ResponsibleForContentIds();
-                if (ctIds != null) pluginApplies = ctIds.Contains(content.Id);
-            }
+                var pluginApplies = plugin.Instance.PluginType() == EPluginType.Global;
+                if (!pluginApplies && plugin.Instance.PluginType() == EPluginType.ContentSpecific)
+                {
+                    var ctIds = plugin.Instance.ResponsibleForContentIds();
+                    if (ctIds != null) pluginApplies = ctIds.Contains(content.Id);
+                }
 
-            if (pluginApplies) plugin.Instance.Patch(content.Id, content.Path.Replace("__data", ""));
+                if (pluginApplies) plugin.Instance.Patch(content.Id, content.Path.Replace("__data", ""));
+            }
+            catch (Exception e)
+            {
+                Logger.Error(e, "Plugin {Name} ({Maintainer}) v{Version} threw an exception while patching {ID} ({path})",
+                                       plugin.Name, plugin.Maintainer, plugin.Version, content.Id);
```

Comment:
Message template has 5 holes, but you're not passing in path.
```suggestion
                                       plugin.Name, plugin.Maintainer, plugin.Version, content.Id, content.Path);

```